index.html

```diff
@@ -179,17 +180,32 @@
         loadData(uri);
         document.getElementById('uriEntered').innerHTML = "You are displaying documents in " + uri.bold();
       }
-      else { //default
+      else {
         loadData('addr.json');
       }
 
+      $("input[name='insert']").serialize(function() {
```

Comment:
Is this actual new code? I think this is left over from Fei's first try at setting up editing. Can it be deleted?